tiledb/api/examples/quickstart_dense.rs

```diff
@@ -6,6 +6,7 @@ use tiledb::Result as TileDBResult;
 const QUICKSTART_DENSE_ARRAY_URI: &str = "quickstart_dense_array";
 const QUICKSTART_ATTRIBUTE_NAME: &str = "a";
 
+/// Returns whether the example array already exists
 fn array_exists() -> bool {
```

Comment:
We should probably nip this pattern in the bud rather than inherit it from core. I'm thinking a `tiledb::Array::exists(&ctx, "some_uri_thing")` or some such?